tests/run.sh

```diff
@@ -13,7 +13,7 @@
 set -e
 
 export SHELL=/bin/sh
-export HOME=/dev/null
+export HOME="${TMP:-/tmp}"
```

Comment:
Might I ask: Why this change? Is it worth adding a comment explaining why need `$HOME` to be a directory?
(I don't really think this matters much, but could this have security implications? Someone else could create `/tmp/.bashrc` and I bet that would end up getting executed)